target/linux/ramips/image/mt7620.mk

```diff
@@ -953,6 +946,19 @@ define Device/sercomm_na930
 endef
 TARGET_DEVICES += sercomm_na930
 
+define Device/sunvalley_loader_okli
```

Comment:
I prefer `sunvalley_filehub` as recipe name, and would include `SOC`, `IMAGE_SIZE` and `DEVICE_PACKAGES` recipe variables too.

Reply:
LOL - I changed this because @mpratt14 preferred this name. Easy for me to change it back, just need folks to agree 🤣. Thanks!

Reply:
Just let Adrian decide, I was following the pattern that he and I decided for my most recent PR

Reply:
Yep, no issue at all - and agreed. Easy change, if needed. Thanks!

Reply:
And my points are to save as many lines as possible, because they differ only in memory size.